src/frontend/typer/behandling/behandlingType.ts

```diff
@@ -9,5 +9,5 @@ export const behandlingTypeTilTekst: Record<BehandlingType, string> = {
     FØRSTEGANGSBEHANDLING: 'Førstegangsbehandling',
     REVURDERING: 'Revurdering',
     TILBAKEKREVING: 'Tilbakekreving',
-    KLAGE: 'Klage',
+    KLAGE: 'Klagebehandling',
```

Comment:
Vi kaller det egentlig bare klage
```suggestion
    KLAGE: 'Klage',
```